main/main.dart

```diff
@@ -0,0 +1,46 @@
+/*
+Status Code :-
+200 :- No error, EveryThing is ok
+205 :- provided data is not appropiate
+206 :- Negative Price or Quantity provided
+207 :- price and quantity should be numeric
+208 :- Something else happens
+*/
+import 'dart:io';
+import 'package:args/args.dart';
+import 'itemTax.dart';
+main(List<String> args) {
+  var parser;
+  parser = ArgParser();
```

Comment:
this can be moved into a util class

Reply:
For an example:
`class BasicUtils{
         static bool isNumeric(String val){ return true;}
}
BasicUtils.isNumeric();
`